keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -12,17 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import tensorflow as tf
```

Comment:
I don't think the ordering of these imports should change, and you are getting errors on CI. You may not be running the format and lint scripts correctly.

https://github.com/keras-team/keras-nlp/runs/6634310403?check_suite_focus=true